doc/ondemand.md

```diff
@@ -115,12 +130,12 @@ The following show how to use the JSON when exceptions are enabled, but simdjson
 support for users who avoid exceptions. See [the simdjson DOM API's error handling documentation](basics.md#error-handling) for more.
 
 * **Extracting Values:** You can cast a JSON element to a native type:
-  `double(element)` or `double x = json_element`. This works for double, uint64_t, int64_t, bool,
-  ondemand::object and ondemand::array. At this point, the number, string or boolean will be parsed,
+  `double(element)` or `double x = json_element`. This works for `double`, `uint64_t`, `int64_t`, `bool`,
```

Comment:
I'm actually intentionally not putting everything into code blocks: code blocks tend to break up the flow of the text in a bad way, and they also end up emphasizing the coded words even more glaringly than bold or italic. Thus, I try not to use code around code terms in a sentence. Full code phrases/statements still need it because you need to be able to see their beginning and end.

I'm still getting a feel for when to use it and when not, but I think some judiciousness is required to keep readability high.

In this case, I think the code doohickeys don't add enough value to offset those negatives. We don't really want to *emphasize* these type words, they are just a list.

Reply:
Change reverted.